serving/docs/lmi/configurations_large_model_inference_containers.md

```diff
@@ -96,7 +96,7 @@ If you are using Neuron container and engine set to Python, the following parame
 
 | Item                           | Required | Description                                                                                                                                                                                                                                                                                                                                           | Example value                                                                                  |
 |--------------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------|
-| option.n_positions             | No       | Input sequence length                                                                                                                                                                                                                                                                                                                                 | Default: `128`                                                                                 |
+| option.n_positions             | No       | Total sequence length, input sequence length + output sequence length.                                                                                                                                                                                                                                                                                | Default: `128`                                                                                 |
```

Comment:
We need consider consolidate with max_rolling_batch_size when rolling batch is enabled.

Reply:
Agreed - I will work on a PR to update the CX, and default behaviors.